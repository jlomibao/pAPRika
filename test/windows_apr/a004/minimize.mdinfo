

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9310E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3492  EGB        =     -127.4407
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7557  RESTRAINT  =        0.0043
 EAMBER  =     -827.4451
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================
