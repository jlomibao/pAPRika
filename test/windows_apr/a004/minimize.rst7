defa
  125
  -0.5276343   4.9589543   1.8308405   1.0088268   4.8944217   1.8318291
  -0.8882590   4.2841636   0.5913063  -0.8941180   4.2465316   3.0472343
  -0.9062785   5.9874797   1.8451961   1.4724401   5.8875291   1.8473980
   1.3134933   4.1923377   0.5926011   1.3129363   4.1529908   3.0480233
   2.6815661   4.0761429   0.1089810   2.6770483   4.0837042   3.5583622
   0.1932449   3.8247765   3.7676014   0.1984365   3.8879553  -0.1415569
   0.1723299   3.3265102   4.8797339   0.1800926   3.4207858  -1.2670953
   2.6665007   4.0304763  -0.9860691   3.2456445   4.9558593   0.4396715
   2.6416365   4.0519686   4.6534357   3.2209755   4.9761014   3.2278942
  -4.6492123   1.9720042   1.8303358  -3.8146656   3.2642319   1.8302898
  -4.2098272   1.2891876   0.6208941  -4.2633666   1.3267164   3.0782462
  -5.7275262   2.1669512   1.8043788  -4.4362551   4.1667393   1.8069225
  -3.0158216   3.1451043   0.6182324  -3.0641162   3.1770043   3.0768928
  -2.2607953   4.2807014   0.1062132  -2.2594558   4.2929374   3.5568940
  -3.3851788   2.0716927   3.8192575  -3.2885911   2.0074173  -0.0951487
  -3.0148528   1.8326469   4.9557667  -2.8537544   1.7266436  -1.1986251
  -2.2483940   4.2313398  -0.9887099  -2.7502659   5.2049563   0.4342574
  -2.2273748   4.2591873   4.6520099  -2.7261511   5.2276496   3.2253109
  -4.0874865  -3.0901661   1.8448318  -4.8083366  -1.7312775   1.8382688
  -3.3246539  -3.0692240   0.6031581  -3.2872186  -3.0368935   3.0608445
  -4.7836136  -3.9365665   1.8661740  -5.8996081  -1.8329084   1.8536527
  -4.3559642  -1.1210442   0.5950140  -4.3227876  -1.0880418   3.0517748
  -4.9291362   0.1279154   0.1134022  -4.9447783   0.1306518   3.5538797
  -3.4682178  -1.8800676   3.7727627  -3.5362208  -1.9372258  -0.1378536
  -3.0167000  -1.6342882   4.8778950  -3.1297216  -1.7261951  -1.2675325
  -4.8849939   0.1372002  -0.9817719  -5.9715665   0.1862866   0.4471367
  -4.9110629   0.1173408   4.6493829  -5.9854900   0.1611301   3.2111889
   0.5805953  -5.1247435   1.8519181  -0.9558744  -5.0594436   1.8528347
   0.9448111  -4.4190953   0.6309767   0.9438830  -4.4402651   3.0853939
   0.9595481  -6.1532205   1.8434477  -1.4206608  -6.0520985   1.8456250
  -1.2606231  -4.3261372   0.6319541  -1.2582091  -4.3458748   3.0863522
  -2.6249407  -4.2537458   0.1246142  -2.6252485  -4.2289103   3.5738880
  -0.1413956  -4.0331809   3.8145983  -0.1423206  -3.9953941  -0.0878811
  -0.1207043  -3.5510425   4.9337540  -0.1221374  -3.4912918  -1.1974560
  -2.5922656  -4.2259905  -0.9706497  -3.1719291  -5.1423703   0.4601615
  -2.6064511  -4.1784928   4.6686719  -3.1891443  -5.1109714   3.2491901
   4.7011851  -2.1349599   1.8369310   3.8677451  -3.4278170   1.8425092
   4.3003584  -1.4864465   0.5954556   4.2729849  -1.4546152   3.0517891
   5.7799668  -2.3285580   1.8507748   4.4895196  -4.3303090   1.8611290
   3.1070081  -3.3397909   0.6025105   3.0766401  -3.3088899   3.0598375
   2.3100115  -4.4605991   0.1226163   2.3162658  -4.4410738   3.5718009
   3.3548897  -2.1721081   3.7725971   3.4116999  -2.2278735  -0.1365118
   2.9254204  -1.8907992   4.8780531   3.0205855  -1.9808182  -1.2643058
   2.2790839  -4.4283388  -0.9725749   2.7807253  -5.3925663   0.4562441
   2.3028252  -4.3910335   4.6666810   2.8027915  -5.3673559   3.2450964
   4.1430242   2.9283288   1.8327891   4.8611920   1.5678433   1.8319208
   3.3396455   2.8808785   0.6188209   3.3858598   2.9045474   3.0782728
   4.8412424   3.7730190   1.8120807   5.9524908   1.6673592   1.8073747
   4.3647508   0.9272775   0.6212749   4.4184116   0.9565355   3.0780679
   4.9796166  -0.2921996   0.1131955   4.9954686  -0.2929899   3.5543674
   3.6050250   1.7728567   3.8179135   3.5093246   1.7232410  -0.0941643
   3.2102476   1.5625401   4.9518522   3.0501046   1.4800682  -1.1966974
   4.9354455  -0.2791732  -0.9819395   6.0234990  -0.3247878   0.4458911
   4.9598233  -0.3032415   4.6498425   6.0353296  -0.3509542   3.2126539
   0.0118265  -0.0767534  -0.0612264  -0.9025512  -0.6618945  -0.2183981
   0.8737357  -0.7362111  -0.2206493   0.0441397   0.7177422  -0.8168157
   0.0384083   0.5183488   1.3579805   0.9468194   1.1201380   1.4838004
  -0.0084829  -0.5833898   2.4361167  -0.8148045   1.1958728   1.4851930
  -0.9174310  -1.1843357   2.3101379   0.8442509  -1.2614464   2.3085317
   0.0189518   0.0098837   3.8558897  -0.8405943   0.6724512   4.0151466
  -0.0170258  -0.7855250   4.6103520   0.9352257   0.5916277   4.0146192
  -0.0000002   0.0002956  -6.0005195  -0.0001257  -0.0007479  -9.0000682
   0.0000300   2.2000034 -11.2000229
