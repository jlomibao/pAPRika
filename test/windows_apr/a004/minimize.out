
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 17:54:28

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/a004
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to    35837 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -8.1356E+02     1.0057E+01     2.4770E+01     N12        57

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -67.7110  EEL     =     1326.2731  EGB        =     -123.8292
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -813.5649
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.2743E+02     5.0058E-01     1.6848E+00     H7        120

 BOND    =        1.2587  ANGLE   =      107.5585  DIHED      =       52.5861
 VDWAALS =      -68.4312  EEL     =     1332.0637  EGB        =     -127.3959
 1-4 VDW =        5.4797  1-4 EEL =    -2130.5477  RESTRAINT  =        0.0013
 EAMBER  =     -827.4281
 NMR restraints: Bond =    0.000   Angle =     0.001   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.2744E+02     4.9621E-01     1.5809E+00     H32       105

 BOND    =        1.2681  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5108  EEL     =     1332.3300  EGB        =     -127.4494
 1-4 VDW =        5.5014  1-4 EEL =    -2130.7376  RESTRAINT  =        0.0034
 EAMBER  =     -827.4423
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.2744E+02     4.9311E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3489  EGB        =     -127.4406
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7555  RESTRAINT  =        0.0043
 EAMBER  =     -827.4450
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.2744E+02     4.9311E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3490  EGB        =     -127.4406
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7555  RESTRAINT  =        0.0043
 EAMBER  =     -827.4450
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.2744E+02     4.9311E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3491  EGB        =     -127.4407
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7556  RESTRAINT  =        0.0043
 EAMBER  =     -827.4450
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.2744E+02     4.9311E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3491  EGB        =     -127.4407
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7556  RESTRAINT  =        0.0043
 EAMBER  =     -827.4450
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.2744E+02     4.9311E-01     1.5809E+00     H32       105

 BOND    =        1.2601  ANGLE   =      107.5457  DIHED      =       52.6104
 VDWAALS =      -68.5195  EEL     =     1332.3490  EGB        =     -127.4406
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7554  RESTRAINT  =        0.0043
 EAMBER  =     -827.4450
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.2744E+02     4.9311E-01     1.5809E+00     H32       105

 BOND    =        1.2601  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3491  EGB        =     -127.4407
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7555  RESTRAINT  =        0.0043
 EAMBER  =     -827.4450
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.2744E+02     4.9311E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3491  EGB        =     -127.4406
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7556  RESTRAINT  =        0.0043
 EAMBER  =     -827.4450
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9310E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3492  EGB        =     -127.4407
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7557  RESTRAINT  =        0.0043
 EAMBER  =     -827.4451
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2744E+02     4.9310E-01     1.5809E+00     H32       105

 BOND    =        1.2602  ANGLE   =      107.5457  DIHED      =       52.6103
 VDWAALS =      -68.5195  EEL     =     1332.3492  EGB        =     -127.4407
 1-4 VDW =        5.5054  1-4 EEL =    -2130.7557  RESTRAINT  =        0.0043
 EAMBER  =     -827.4451
 NMR restraints: Bond =    0.000   Angle =     0.003   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.302    9.302    0.000    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.347  147.398    0.051    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.481   49.483    0.002    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   61.988   62.003    0.014    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.144 -146.178    0.034    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.650  -72.548    0.102    0.000 t
  DUM  DM1    3 --  C    BUT    2:    5.940    6.000    0.060    0.000 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.229  180.000    0.771    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  177.990  180.000    2.010    0.003 a
                                       Total distance penalty:      0.000
                                       Total angle    penalty:      0.003
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.465
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.37 (22.66% of Gen B)
|                Calc gb diag               4.13 (39.47% of Gen B)
|                Calc gb off-diag           3.96 (37.84% of Gen B)
|                Other                      0.00 ( 0.03% of Gen B)
|             Gen Born time             10.47 (100.0% of Nonbo)
|          Nonbond force             10.47 (90.67% of Force)
|          Bond/Angle/Dihedral        1.04 ( 9.01% of Force)
|          Other                      0.04 ( 0.32% of Force)
|       Force time                11.55 (100.0% of Runmd)
|    Runmd Time                11.55 (99.78% of Total)
|    Other                      0.03 ( 0.22% of Total)
| Total time                11.57 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 17:54:28.031  on 04/10/2019
|           Setup done at 17:54:28.037  on 04/10/2019
|           Run   done at 17:54:39.604  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
